trueos-handbook/introducing.rst

```diff
@@ -718,55 +770,104 @@ A virtualized environment allows a user to test drive an operating
 system without overwriting the current operating system. This is an
 excellent way to practice installation, determine whether the hardware
 is supported, or to try multiple versions of different operating
-systems. Virtualization software effectively creates windows (known as
-virtual machines) to install and use an operating system. The only
+systems. Virtualization software creates a virtual machine, which is
+essentially a computer environment created entirely in software, that
+allows the installation and use of an operating system. The only
 limitation to virtualization is the hardware, as each virtual machine
-uses CPU and RAM. Depending upon the amount of CPU and RAM in the
-computer, the installed operating system using virtualization software
-may run slowly. If the computer slows down, try closing other
-applications running on the computer to free up some RAM.
-
-To run virtualization software on |trueos|, search for *virtualbox*
-within the |sysadm| :sysclbk:`AppCafe <appcafe>` and install the
-`VirtualBox <https://www.virtualbox.org/>`_ open source virtualization
-program and the
-`VirtualBox Guest Additions <http://www.virtualbox.org/manual/ch04.html>`_.
+uses real resources, the CPU and RAM. Depending on the amount of CPU and
+RAM available on the host computer, the operating system installed as a
+guest in the virtual environment may run slowly. If the host computer
+slows down, closing other non-essential applications to free up CPU/RAM
+may help.
+
+.. index:: bhyve
+.. _bhyve:
+
+bhyve
+-----
+
+bhyve (pronounced bee hive) is a type-2 hypervisor that runs on
+natively on |trueos| and originally developed on FreeBSD. bhyve runs
+FreeBSD 9+, OpenBSD, NetBSD, Linux and Windows NT guests. Current
+development efforts aim at widening support for other operating systems
+for the x86-64 architecture.
+The `Virtualization <https://www.freebsd.org/doc/handbook/virtualization-host-bhyve.html>`_
+section of the FreeBSD Handbook has in-depth instructions about bhyve
+features and use. bhyve, while very powerful, is not currently very user
+friendly. bhyve is under active development with many great features on
+the way.
+
+For a more user-friendly virtualization experience, many users prefer
+:ref:`VirtualBox`.
+
+.. TODO: Add additional information for using bhyve
+
+.. index:: virtualbox
+.. _VirtualBox:
+
+VirtualBox
+----------
+
+VirtualBox is a popular virtualization software available in |trueos|.
+Installing VirtualBox through the |sysadm| :sysclbk:`AppCafe <appcafe>`
+GUI, or :command:`pkg install virtualbox-ose`  on the command line, will
```

Comment:
"`AppCafe <appcafe>` GUI, or :command:`pkg install virtualbox-ose`  on the command line, will ..."
->
"`AppCafe <appcafe>` or typing :command:`pkg install virtualbox-ose` on the command line will ..."

(Note commas and double-space removals).